src/tito/tagger/main.py

```diff
@@ -243,10 +239,21 @@ def _make_changelog(self):
 
                 write(fd, header)
 
-                for cmd_out in output.split("\n"):
-                    write(fd, "- ")
-                    write(fd, "\n  ".join(textwrap.wrap(cmd_out, 77)))
+                # don't die if this is a new package with no history
+                if self._changelog_msg is not None:
+                    write(fd, self._changelog_msg)
```

Comment:
Would you be willing to go to --changelog "line1" --changelog "line2" etc? I'm not crazy about a --changelog "- line1\n- line2". It's fine programatically but very awkward for normal CLI user. I'd prefer this, or --changelog-file for programatic usage.